numpy/lib/type_check.py

```diff
@@ -262,6 +265,10 @@ def isreal(x):
     out : ndarray, bool
         Boolean array of same shape as `x`.
 
+    Notes
+    --------
```

Comment:
Underline needs to be the same length as the title.

Reply:
fixed!